modules/apps/item-selector/item-selector-web/src/main/java/com/liferay/item/selector/web/internal/portlet/ItemSelectorPortlet.java

```diff
@@ -69,7 +74,25 @@ public void render(
 
 		localizedItemSelectorRendering.store(renderRequest);
 
-		super.render(renderRequest, renderResponse);
+		try {
+			PermissionChecker permissionChecker =
```

Comment:
Don't we have the permission checker in the ThemeDisplay?

Reply:
Changed to themeDisplay based.